pkg/netdevice/pciNetDevice.go

```diff
@@ -33,6 +33,8 @@ type pciNetDevice struct {
 }
 
 // NewPciNetDevice returns an instance of PciNetDevice interface
+//
+//nolint:gocyclo
```

Comment:
do we still need this ? function is not changed now :)

Reply:
you are right - done